### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IvanGuculyak**

### CODE REVIEW

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
you didn't add all of files - models, views and others

File: db.sqlite3
Comment (Grayder0152) [inline db.sqlite3:1]: Remove this file from repo

**Review by Grayder0152 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.pytest_cache/
+**__pycache__/
+**db.sqlite3
```
Comment (ArsenPidhoretskyi) [inline .gitignore:9]: Add empty line

File: cinema/views.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+
+        if serializer.is_valid():
```
Comment (ArsenPidhoretskyi) [inline cinema/views.py:20]: Use serializer.is_valid(raise_exception=True) instead of if statement
Fix in all places

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: cinema/migrations/__pycache__/0001_initial.cpython-311.pyc
Comment (Grayder0152) [inline cinema/migrations/__pycache__/0001_initial.cpython-311.pyc:1]: Add __pycache__ to gitignore

File: cinema/views.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        serializer.save()
+        return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+@api_view(["GET", "PUT", "DELETE"])
+def movie_detail(request, pk):
+    try:
+        movie = Movie.objects.get(pk=pk)
+    except Movie.DoesNotExist:
+        return Response(status=status.HTTP_404_NOT_FOUND)
```
Comment (vsmutok) [inline cinema/views.py:30]: use `get_object_or_404`

**Review by vsmutok — APPROVED** [review]
Looks Good 👍

**Pull request closed without merging**